src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,240 @@
+  pedantic 1.8.0 (1.9.0 available)
+Changed 1 dependency!
+```
+
+To fix the second dependency (`pedantic`),
+you can change the pubspec's `pedantic` entry to use [caret syntax][] (`^`)
+and the number in the **Resolvable** column: **`^1.9.0`**.
+Here's the diff for `pubspec.yaml`:
+
+{% comment %}
+  [TODO: Improve the formatting of the following diff]
+{% endcomment %}
+
+```diff
+-  pedantic: 1.8.0
++  pedantic: ^1.9.0
+```
+
+{{site.alert.note}}
+  Updating the `pedantic` package can cause code to start failing analysis,
+  depending on your code and how you include the analysis options file.
+  For details, see the section on
+  [enabling default Google rules][using pedantic]
+  in [Customizing static analysis][].
+{{site.alert.end}}
+
+After editing `pubspec.yaml`, run `pub upgrade` to
+update the `pubspec.lock` file.
+You can then run `pub outdated` to confirm that
+you've made all necessary changes.
+
+```terminal
+$ pub upgrade
+...
+$ pub outdated
+Found no outdated packages.
+```
+
+{{site.alert.important}}
+  Test your code to verify that it still works as expected
+  after updating the packages.
+{{site.alert.end}}
+
+
+## Output columns
+
+The output of `pub outdated` has four columns of version information
+for each out-of-date dependency:
+
+```terminal
+$ pub outdated
+Dependencies  Current  Upgradable  Resolvable  Latest  
+args          1.5.0    1.6.0       1.6.0       1.6.0   
+
+dev_dependencies
+pedantic      1.8.0    1.8.0       1.9.0       1.9.0   
+
+transitive dependencies: all up-to-date
+
+transitive dev_dependencies: all up-to-date
+
+1 upgradable dependency is locked (in pubspec.lock) to an older version.
+To update it, use `pub upgrade`.
+
+1 dependency is constrained to a version that is older than a resolvable version.
+To update it, edit pubspec.yaml.
+```
+
+Here are the meanings of each column:
+
+Current
+: The version used in your package, as recorded in `pubspec.lock`.
+  If the package isn't in `pubspec.lock`,
+  the value is `-`.
+
+Upgradable
+: The latest version allowed by your `pubspec.yaml` file.
+  This is the version that `pub upgrade` resolves to.
+  The value is `-` if the value in the **Current** column is `-`.
+
+Resolvable
+: The latest version that can be resolved,
+  when combined with all other dependencies.
+  This version corresponds to what you get if
+  you set all version constraints in `pubspec.yaml` to `any`
+  and then run `pub upgrade`.
+  A value of `-` means that the package won't be needed.
+
+{{site.alert.important}}
+  We don't recommend actually using `any` as the constraint,
+  because then running `pub upgrade` might surprise you by making
+  incompatible version changes.
+{{site.alert.end}}
+
+Latest
+: The latest version of the package available,
+  excluding prereleases unless you use the option `--prereleases`.
+
+For example, say your app depends on the `foo` and `bar` packages,
+but the latest version of `bar` depends on an old major version of `foo`.
```

Comment:
Maybe change "latest version of `bar` depends on an old major version of `foo" to something like "latest version of `bar` allows only older major versions of `foo" ?

Reply:
That seems a little harder to read. Is it important to talk about multiple major versions?